controllers/operator_controller.go

```diff
@@ -20,6 +20,9 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/go-logr/logr"
+	operatorv1 "github.com/operator-framework/api/pkg/operators/v1"
```

Comment:
Why are we pulling in the `Operator` from `operator-framework/api` (OLMv0) rather than using the one included in this repo (OLMv1)? (i.e. `operatorsv1alpha1` below?) Are we conflating the two Operator CRDs?

Reply:
you're right, this should be `operatorsv1alpha1`, thanks!